### PR TITLE
add parent info

### PR DESCRIPTION
**Pull request by supun-ebi**

### CODE REVIEW

File: uniprotkb-rest/src/main/java/org/uniprot/api/uniprotkb/groupby/service/GroupByECService.java [hunk 1]
```diff
@@ -11,6 +8,9 @@
 import org.uniprot.api.uniprotkb.service.UniProtEntryService;
 import org.uniprot.core.cv.ec.ECEntry;
 
+import java.util.*;
```
Comment (LeonardoGonzales) [inline uniprotkb-rest/src/main/java/org/uniprot/api/uniprotkb/groupby/service/GroupByECService.java:11]: Can you please run maven command to make sure that import order is based on maven plugin?

File: uniprotkb-rest/src/test/java/org/uniprot/api/uniprotkb/groupby/service/GroupByECServiceTest.java [hunk 2]
```diff
@@ -28,6 +16,18 @@
...
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.empty;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+import static org.uniprot.api.uniprotkb.groupby.service.GroupByECService.*;
+
 @ExtendWith(MockitoExtension.class)
 class GroupByECServiceTest {
```
Comment (LeonardoGonzales) [inline uniprotkb-rest/src/test/java/org/uniprot/api/uniprotkb/groupby/service/GroupByECServiceTest.java:32]: Excellent, the rename!!

File: uniprotkb-rest/src/test/java/org/uniprot/api/uniprotkb/groupby/service/go/GOServiceTest.java [hunk 3]
```diff
@@ -27,11 +29,21 @@ void setUp() {}
...
 
         List<GoRelation> children = goService.getChildren(ID);
 
         verify(goClient).getChildren(ID);
-        assertSame(result, children);
+        assertSame(resultChildren, children);
+    }
+
+    @Test
+    void getGoRelation() {
```
Comment (LeonardoGonzales) [inline uniprotkb-rest/src/test/java/org/uniprot/api/uniprotkb/groupby/service/go/GOServiceTest.java:41]: Do you think a test for not found should be added? 
Do you think the same would also apply for getChildren?

Comment (supun-ebi) [reply uniprotkb-rest/src/test/java/org/uniprot/api/uniprotkb/groupby/service/go/GOServiceTest.java:41]: I don't think we need separate tests since empty list and empty optional scenarios are covered in the written tests. (mocked results can get one of any form)
